### PR TITLE
Git 4282

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by yanan** [comment]
ft retest

File: tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/MovedStagingProperties.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ */
+@Component(
+	property = {
+		"file.extensions=properties", "problem.title=Moved Two Staging Properties to OSGi Configuration",
+		"problem.summary=Two Staging properties have been moved from portal.properties to an OSGi configuration named ExportImportServiceConfiguration.java in the export-import-service module.",
+		"problem.tickets=LPS-88018", "problem.section=#moved-staging-properties", "version=7.2"
+	},
+	service = FileMigrator.class
+)
+public class MovedStagingProperties extends PropertiesFileMigrator {
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/MovedStagingProperties.java]: Hey @Seiphon  can you send a test for this migrator?  @jtydhr88 can you help him with how to write tests for these migrators?

File: tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/RemoveLinkApplicationURLs.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ */
+@Component(
+	property = {
+		"file.extensions=java,jsp,jspf", "problem.title=Remove Link Application URLs to Page Functionality",
+		"problem.summary=The Link Portlet URLs to Page option in the Look and Feel portlet was marked as deprecated in Liferay Portal 7.1, allowing the user to show and hide the option through a configuration property. In Liferay Portal 7.2, this has been removed and can no longer be configured.",
+		"problem.tickets=LPS-85948", "problem.section=#remove-link-application-urls-to-functionality", "version=7.2"
+	},
+	service = FileMigrator.class
+)
+public class RemoveLinkApplicationURLs extends JavaFileMigrator {
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/RemoveLinkApplicationURLs.java]: Need a test for this as well

File: tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/BREAKING_CHANGES.markdown [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+This affects anyone with a value for `dl.tabs.visible` different than `false`.
+
+#### How should I update my code?
+
+No code changes are necessary.
+
+#### Why was this change made?
+
+Documents & Media has been reviewed from an UX perspective, and removing the
+navigation tabs in widget pages was part of a UI clean up process.
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/BREAKING_CHANGES.markdown]: @Seiphon  can you create issues on github for all of the breaking changes checkers that we need to write to cover the 7.2 breaking changes?

Comment (gamerson) [reply tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/BREAKING_CHANGES.markdown]: @jtydhr88 you can target them to the m3 milestone

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gamerson** [comment]
Merged in 0e4b57e61 .  cc @jtydhr88

File: tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/BREAKING_CHANGES.markdown (hunk 3 above)
Comment (Seiphon) [reply tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/liferay72/BREAKING_CHANGES.markdown]: sure, will do that

**Pull request closed without merging**